tests/test_memory.py

```diff
@@ -69,6 +69,7 @@ def __init__(self):
 
         # Create a memory gateway
         self.ms = rogue.interfaces.memory.TcpServer("127.0.0.1",9080);
+        time.sleep(1)
```

Comment:
Instead of a `time.sleep()`, can the `rogue.interfaces.memory.TcpServer()` have a variable that you can poll to know when the link is up?  Example of what I would use instead of time.sleep
```
while (self.ms.LinkUp != 0x0)
```

Reply:
In reality this should not be necessary.